cpp/include/experimental/graph_view.hpp

```diff
@@ -140,27 +140,29 @@ class partition_t {
     auto major_first =
       hypergraph_partitioned_
         ? vertex_partition_offsets_[row_comm_size_ * partition_idx + row_comm_rank_]
-        : vertex_partition_offsets_[row_comm_rank_ * col_comm_size_];
+        : vertex_partition_offsets_[col_comm_rank_ * row_comm_size_];
     auto major_last =
       hypergraph_partitioned_
         ? vertex_partition_offsets_[row_comm_size_ * partition_idx + row_comm_rank_ + 1]
-        : vertex_partition_offsets_[(row_comm_rank_ + 1) * col_comm_size_];
+        : vertex_partition_offsets_[(col_comm_rank_ * row_comm_size_)+1];
```

Comment:
This should be (col_comm_rank_ + 1) * row_comm_size

Reply:
done